src/components/App.tsx

```diff
@@ -999,6 +1027,42 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     }
   }
 
+  private calculateNetStats = () => {
+    this.checkNetworkPing();
+    this.checkNetworkSpeed();
+  };
+
+  private checkNetworkPing = async () => {
+    if (!this.state.showStats || !this.props.isCollaborating) {
+      clearTimeout(this.networkPingIntervalId);
+      return;
+    }
+    const networkPing = await simulatePing();
+    this.setState({ networkPing });
+    if (this.networkPingIntervalId) {
+      clearTimeout(this.networkPingIntervalId);
+    }
+    this.networkPingIntervalId = setTimeout(
+      this.checkNetworkPing,
+      NETWORK_SPEED_TIMEOUT_MS,
```

Comment:
lets rename to `NETWORK_TIMEOUT_MS`if we have the same timeout for ping and speed